entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -718,23 +718,37 @@ class EntityService(
         neo4jRepository.deleteEntity(entityId)
 
     @Transactional
-    fun deleteEntityAttribute(entityId: String, attributeName: String, contextLink: String): Boolean {
+    fun deleteEntityAttributes(entityId: String, attributeName: String, contextLink: String): Boolean {

```

Comment:
This function name (`deleteEntityAttributes`) and name of the following (`deleteEntityAttribute`) do not clearly expose the intent of the function.

For instance, `deleteEntityAttributes` delete one attribute, so the plural form of attribute is misleading.

I would suggest:
- deleteEntityAttribute for this function (it deletes an attribute and all of its instances if it has some)
- deleteEntityAttributeInstance for the following one (it only deletes an instance of an attribute)